core/core.go

```diff
@@ -26,31 +26,29 @@ package core
 
 import (
 	"context"
+	"github.com/spf13/viper"
 	"time"
 
+	"fmt"
+	"github.com/AliceO2Group/Control/common/logger"
+	"github.com/looplab/fsm"
 	"github.com/mesos/mesos-go/api/v1/lib/extras/scheduler/callrules"
 	"github.com/mesos/mesos-go/api/v1/lib/extras/store"
 	"github.com/mesos/mesos-go/api/v1/lib/scheduler"
-	"github.com/looplab/fsm"
 	"github.com/sirupsen/logrus"
-	"github.com/AliceO2Group/Control/common/logger"
-	"fmt"
 	"net"
 )
 
 var log = logger.New(logrus.StandardLogger(),"core")
 
 // Run is the entry point for this scheduler.
 // TODO: refactor Config to reflect our specific requirements
-func Run(cfg Config) error {
-	if cfg.veryVerbose {
-		cfg.verbose = true
-	}
-	if cfg.verbose {
+func Run() error {
+	if viper.GetBool("verbose") || viper.GetBool("veryVerbose") {
```

Comment:
Is it still the case that `veryVerbose true` also sets `verbose` to `true`?
Generally this `verbose` and `veryVerbose` stuff needs to be sorted out, unless we want to end up with `veryVeryVerbose` and `superDuperOmgWtfBbqVerboseAF` sometime down the line.
For now let's just make sure that `veryVerbose` implies `verbose` to preserve existing behavior.
For more information on future developments, see [OCTRL-68](https://alice.its.cern.ch/jira/browse/OCTRL-68)